spec/controllers/v0/institutions_controller_spec.rb

```diff
@@ -224,8 +236,9 @@
     end
 
     it 'returns profile details' do
-      school = create(:institution, :in_chicago)
-
+      school = create(:institution, :in_chicago, approved: true)
+      # adding a non approved institutions row
+      create(:institution, :contains_harv, approved: false)
```

Comment:
You can remove lines 240 and 241 because they do not affect the retrieving of the details of the school created on line 239.

Reply:
Sure, both lines have been removed.